extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/TlsTrustManagersProviderConfig.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.amazon.common.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class TlsTrustManagersProviderConfig {
+
+    // @formatter:off
+    /**
+     * TLS trust managers provider type.
+     *
+     * Available providers:
+     *
+     * * `insecure` - Use this provider if you don't want to validate the servers certificates.
```

Comment:
maybe `trust-all` would be a better name, wdyt ?

Reply:
Trusting all certificates is generally a bad thing to do, thats why i choose a "negative" name. But i can change it to "trust-all" if you prefer it.

Reply:
I'm a bit late to the party but let's go with `trust-all`. It's more consistent with the other parts of the code.